types/node-tlv/node-tlv-tests.ts

```diff
@@ -0,0 +1,116 @@
+import TLV from 'node-tlv';
+import assert from 'assert';
```

Comment:
```suggestion
import TLV = require('node-tlv');
import assert = require('assert');
```
as per previosu comment about `esModuleInterop`